scripts/install

```diff
@@ -277,11 +277,11 @@ yunohost firewall allow --no-upnp UDP 67
 # Set default inits
 # The boot order of these services are important, so they are disabled by default
 # and the ynh-hotspot service handles them.
-systemctl disable hostapd
-systemctl stop hostapd
-systemctl enable php5-fpm
-systemctl restart php5-fpm
-systemctl reload nginx
+yunohost service disable hostapd
+yunohost service stop hostapd
+yunohost service enable php5-fpm
+yunohost service restart php5-fpm
+yunohost service reload nginx
```

Comment:
Uuuuh, I think for regular service we should keep the regular `systemctl` thing ? :sweat_smile: